x/incentives/keeper/distribute_test.go

```diff
@@ -241,6 +248,9 @@ func (suite *KeeperTestSuite) TestNoLockNonPerpetualGaugeDistribution() {
 	suite.Require().NoError(err)
 
 	// distribute coins to stakers, since it's perpetual distribute everything on single distribution
+	// TODO: its not perpetual so this shouldnt be true. I can fix this once I understand
+	// what both TestNoLockPerpetualGaugeDistribution and TestNoLockNonPerpetualGaugeDistribution
+	// are actually trying to prove.
 	distrCoins, err := suite.App.IncentivesKeeper.Distribute(suite.Ctx, []types.Gauge{*gauge})
```

Comment:
TODO: its not perpetual so this shouldnt be true. I can fix this once I understand what both TestNoLockPerpetualGaugeDistribution and TestNoLockNonPerpetualGaugeDistribution are actually trying to prove.

Reply:
Both `TestNoLockPerpetualGaugeDistribution` and `TestNoLockNonPerpetualGaugeDistribution` are testing the case where there are no locks for a gauge to distribute to when `suite.App.IncentivesKeeper.Distribute` is called. The expected behavior is that no tokens are distributed at all, which is why two lines later it makes sure that the number of distributed coins is zero:
```
suite.Require().Equal(distrCoins, sdk.Coins(nil))
```